frontend/www/js/omegaup/arena/clarifications.ts

```diff
@@ -3,21 +3,49 @@ import * as time from '../time';
 import { types } from '../api_types';
 import clarificationStore from './clarificationsStore';
 
-export interface ClarificationEvent {
+export interface ContestClarification {
   contestAlias: string;
   clarification: types.Clarification;
 }
 
-export function refreshContestClarifications({
-  contestAlias,
-  problemAlias,
-}: {
+export enum ContestClarificationType {
+  WithProblem,
+  AllProblems,
+}
+
+interface ContestClarificationWithProblem {
+  type: ContestClarificationType.WithProblem;
   contestAlias: string;
-  problemAlias?: string;
-}) {
+  problemAlias: string;
+}
+
+interface ContestClarificationAllProblems {
+  type: ContestClarificationType.AllProblems;
+  contestAlias: string;
+}
+
+type contestClarificationRequest =
```

Comment:
```suggestion
type ContestClarificationRequest =
```

oops, no me di cuenta de esto. todos los tipos van en mayúscula.

Reply:
Listo.